### PR TITLE
feat(unlock-app) - customize recipients for withdrawal

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

**Modal** 

Screenshot 2023-02-09 at 17 13 41

Screenshot 2023-02-09 at 17 17 11

**Form validations** 

Screenshot 2023-02-02 at 12 55 10

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
For the address can we put the address of the current user by default? and call that `beneficiary`?
For the amount let's use a `<input type="range">` between 0 and the current balance and add a warning indicating that if the user withdraws too much users would not be able to cancel their memberships.

File: unlock-app/src/components/interface/locks/Manage/elements/WithdrawFundModal.tsx [hunk 1]
```diff
@@ -101,23 +111,26 @@ export const WithdrawFundModal = ({
...
+            {...register('beneficiary', {
               required: {
                 value: true,
                 message: 'This field is required.',
               },
               minLength: {
                 value: 3,
-                message: 'Address should be 3 characters long at least.',
+                message:
+                  'Beneficiary address should be 3 characters long at least.',
```
Comment (julien51) [inline unlock-app/src/components/interface/locks/Manage/elements/WithdrawFundModal.tsx:122]: That's not correct. it should be an ethereum address or ENS. Please use the component that we have that supports that?

Comment (kalidiagne) [reply unlock-app/src/components/interface/locks/Manage/elements/WithdrawFundModal.tsx]: We don't actually have the component merged, I felt a comment to replace it when it is ready!

**Comment by kalidiagne** [comment]
> For the address can we put the address of the current user by default? and call that `beneficiary`? For the amount let's use a `<input type="range">` between 0 and the current balance and add a warning indicating that if the user withdraws too much users would not be able to cancel their memberships.

Updated 

Screenshot 2023-02-02 at 18 54 37

**Review by searchableguy — COMMENTED** [review]
Can remove the top notification far since we have a loading button with text? WDYT? less clutter.

**Comment by kalidiagne** [comment]
> Can remove the top notification far since we have a loading button with text? WDYT? less clutter.

Updated!

**Comment by kalidiagne** [comment]
@julien51 added `AddressInput` component

**Review by julien51 — APPROVED** [review]

**Comment by julien51** [comment]
LG!

**Pull request merged**